clientd3d/d3drender_materials.h

```diff
@@ -0,0 +1,57 @@
+// Meridian 59, Copyright 1994-2024 Andrew Kirmse and Chris Kirmse.
+// All rights reserved.
+//
+// This software is distributed under a license that is described in
+// the LICENSE file that accompanies it.
+//
+// Meridian is a registered trademark.
+//
+// This source file contains a collection of functions responsible for configuring and managing 
+// various materials and rendering states within a Direct3D - based rendering engine.
+//
+#ifndef _D3DRENDERMATERIALS_H
+#define _D3DRENDERMATERIALS_H
+
+// material functions
+Bool D3DMaterialNone(d3d_render_chunk_new* pPool);
+
+// world
+Bool D3DMaterialWorldPool(d3d_render_pool_new* pPool);
```

Comment:
Can you disable your auto-formatter?  These files now have a different syntax convention than the rest of the codebase (d3d_render_pool_new* pPool instead of d3d_render_pool_new *pPool).